package.json

```diff
@@ -26,9 +26,9 @@
   "scripts": {
     "postinstall": "npm run postinstall:frontend && npm run postinstall:shared",
     "test": "npm run test:backend && npm run test:frontend",
-    "test:backend": "env-cmd -f __tests__/setup/.test-env jest",
-    "test:backend:ci": "env-cmd -f __tests__/setup/.test-env jest --maxWorkers=2 --logHeapUsage --workerIdleMemoryLimit=0.2",
-    "test:backend:watch": "env-cmd -f __tests__/setup/.test-env jest --watch",
+    "test:backend": "env-cmd -f __tests__/setup/.test-env node --experimental-vm-modules node_modules/jest/bin/jest.js",
```

Comment:
Trying to understand this change - why did we need to add this experimental-vm-modules flag?

Reply:
`react-email@v1` introduced the use of dynamic imports. i.e., at runtime doing `requires('somePackage')`. This is something that jest isn't very happy with.

For non-jest usage, we're running them through a different bundler / transpiler. While for tests, we're directly running it through jest.

Reply:
for more context, react email v3.0.2 does an ES modules dynamic import syntax `const { default: reactDOMServer } = await import("react-dom/server");`. Since we are using jest, it does not support ES module syntax out of the box and requires the `experimental-vm-modules` flag to support this ESM syntax